.github/workflows/ci.yml

```diff
@@ -0,0 +1,79 @@
+name: CI
+
+on:
+  push:
+    branches:
+      - main
+    tags:
+      - v*
+  pull_request:
+    branches:
+      - "*"
+  schedule:
+    # Daily at 06:14
+    - cron: '14 6 * * *'
```

Comment:
Any reason for this exact time? Do we really need to it run daily anyways?

Reply:
Each time I copy this I shift the start time a bit just to stagger.  I probably don't stagger sufficiently but...  The general intent is sometime in the night.  This project probably benefits less from nightly builds but it is nice to know when things broke and building more often makes it more likely you will know which one thing is the cause of a break.  But, meh, not a big deal.

Reply:
It might be best to at least comment that the exact time has no significance, so that it doesn't confuse someone in the future